pkg/http/request_log.go

```diff
@@ -156,6 +157,8 @@ func (h *RequestLogHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 }
 
 func (h *RequestLogHandler) write(t *template.Template, in *RequestLogTemplateInput) {
+	h.writerMux.Lock()
```

Comment:
So, what if instead we write to a local buffer and then just dump the local buffer into the `h.writer`. Template execution is not very fast and I'd prefer not to lock over it.

For the actual setup we're using `pkg/logging/sync_file_writer.go#L35`, which does this locking itself. So as long as the whole template expansion is written this will be safe

Reply:
Good point. Done.